pages/admin/courses/_id.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="container">
+    <form>
+      <vue-form-generator
+        :schema="schema"
+        :model="model"
+        :options="formOptions"
+      >
+      </vue-form-generator>
+    </form>
+  </div>
+</template>
+ 
+<script>
+import Vue from 'vue'
+const VueFormGenerator = require('vue-form-generator')
+import 'vue-form-generator/dist/vfg.css'
+
+Vue.use(VueFormGenerator)
```

Comment:
This code belong in a plugin. See https://github.com/coding-blocks/amoeba-frontpages/blob/master/online.codingblocks.com/plugins/vuency.js